handlers.go

```diff
@@ -206,6 +206,7 @@ func (a *autographer) handleSignature(w http.ResponseWriter, r *http.Request) {
 			"ref":        sigresps[i].Ref,
 			"type":       sigresps[i].Type,
 			"signer_id":  sigresps[i].SignerID,
+			"key_id":     sigreq.KeyID,
```

Comment:
I'm confused. Under which circumstances would the keyid be different from the signerid?

Reply:
I think we were trying to debug signer resolution for the `-` vs `_` csigpki discrepancy.

I mostly want to associate signing HTTP request param with each requested signature in that HTTP request.